frontend/templates/product/sections/ProductSection/index.tsx

```diff
@@ -259,9 +259,12 @@ const ProductTitle = chakra(
 
 type CustomAccordionButtonProps = React.PropsWithChildren<{}>;
 
-function CustomAccordionButton({ children }: CustomAccordionButtonProps) {
+function CustomAccordionButton({
+   children,
+   ...other
```

Comment:
Does the `other` represent the `data-testid` ? If so, were there no other ways to select the `AccordionButton` like `getByRole`?

Reply:
There are multiple of these AccordionButtons on the page so we need some sort of id to get the specific one.

Reply:
As we write more tests, I am seeing an increase in just defaulting to adding the `data-testid` attribute to elements, but I am not sure if that is a good practice.

Would using the `name` option to filter out the button work?

```
getByRole('button', { name: 'Compatibility' }). 
```

Or additional using `page.$$`?
```
const compatibleDevicesDropwdown = await page.$eval('button', (button) => {
  if (button.textContent === 'Compatible Devices') {
    return button;
  }
});
```

<img width="745" alt="Submit button@2x" src="https://user-images.githubusercontent.com/22064420/207940951-c58a79bf-ea11-4ba1-a5a3-322d4519563a.png">